src/app/common/services/ai-skills.service.ts

```diff
@@ -12,34 +12,35 @@ export class AiSkillsService extends BaseHttpApiService {
 		protected loginService: SessionService,
 		protected http: HttpClient,
 		protected configService: AppConfigService,
-		protected messageService: MessageService
+		protected messageService: MessageService,
 	) {
-        super(loginService, http, configService, messageService);
-    }
+		super(loginService, http, configService, messageService);
+	}
 
-    getAiSkillsResult(textToAnalyze: string): Promise<AiSkillsResult> {
-        // TODO: Potentially it would be better to transfer the text to analyze in the body,
-        // since it could become quite long. For this howerver, POST needs to be used.
-        // This either (probably) failed with some CSRF stuff (though there was no clear error message
-        // indicating this) or the authentication had to be removed (that's how it was done for uploading image).
-        // Removing authentication could lead to attacks though and I didn't manage to fix the CSRF stuff
-        // (I've tried for hours to add the correct headers etc., it all failed with a 403 without an error message).
-        // Since the url lenght limit *probably* isn't too short anyway, for now this solution should suffice.
-        return this
-            .get<AiSkillsResult>(`/aiskills/${btoa(textToAnalyze)}`)
```

Comment:
It might make sense to review the [fix of the AI tool](https://github.com/orgs/mint-o-badges/projects/2/views/1?pane=issue&itemId=59011637) first and merge it into this branch, because currently this wouldn't work

Reply:
I adjusted that